driver/controller.go

```diff
@@ -127,27 +115,22 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		}
 
 		ll.Info("volume already created")
-		csiVolume.Id = vol.UUID
+		csiVolume.VolumeId = vol.UUID
 		return &csi.CreateVolumeResponse{Volume: &csiVolume}, nil
 	}
 
 	volumeReq := &cloudscale.Volume{
-		/* Region:        d.region, */
```

Comment:
Please reinstate this. We will start using regions very soon. It's a good reminder that this is not finished.

Reply:
I reintroduced it; I wasn't aware that support for multiple regions is going to be available soon